extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/unmarshaler.go

```diff
@@ -356,12 +356,57 @@ func (f *elbAccessLogUnmarshaler) addToNLBAccessLogs(resourceAttr *resourceAttri
 	recordLog.Attributes().PutInt(string(conventions.HTTPRequestSizeKey), nlbRecord.ReceivedBytes)
 	recordLog.Attributes().PutInt(string(conventions.HTTPResponseSizeKey), nlbRecord.SentBytes)
 	recordLog.Attributes().PutStr(AttributeTLSListenerResourceID, nlbRecord.Listener)
-	recordLog.Attributes().PutStr(string(conventions.TLSProtocolVersionKey), nlbRecord.TLSProtocolVersion)
-	recordLog.Attributes().PutStr(string(conventions.TLSCipherKey), nlbRecord.TLSCipher)
+	recordLog.Attributes().PutInt(AttributeELBConnectionTime, nlbRecord.ConnectionTime)
+	recordLog.Attributes().PutInt(AttributeELBTLSHandshakeTime, nlbRecord.TLSHandshakeTime)
+	recordLog.Attributes().PutStr(AttributeELBTLSConnectionCreationTime, nlbRecord.TLSConnectionCreationTime)
+
+	// Attributes below may be unset (set to "-") in logs
+
+	if nlbRecord.IncomingTLSAlert != unknownField {
+		recordLog.Attributes().PutStr(AttributeELBIncomingTLSAlert, nlbRecord.IncomingTLSAlert)
+	}
```

Comment:
So many of these conditions, I would just add a new function `addStr` that makes this check, and use it everywhere

Reply:
It's common for all signal under ELB: nlb, clb & alb. And this will be a small refactoring which IMO should be done in a dedicated PR.